pkg/wait/conditions.go

```diff
@@ -31,11 +31,11 @@ type Range struct {
 }
 
 // WaitCondition is a function performing a condition check for if we need to keep waiting
-type WaitCondition func() (done bool, err error)
+type WaitCondition func() (done bool, err error) // nolint
```

Comment:
What is the lint failure here?

Reply:
`wait.WaitCondition` was mentioned as stuttering 😅

Reply:
ah! Yeah, ignore 😆